src/utils/CLIENT_DEFAULTS.js

```diff
@@ -9,20 +9,20 @@ import { cloneDeep } from 'lodash';
  */
 const CLIENT_DEFAULTS = {
   
-  // Property Name                | Default Value   | Valid Value Description
+  // Property Name                | Default Value    | Valid Value Description
 
-  // @todo get this value from the app somewhere  
-  USState: 'MA',                   // Two-letter code denoting the state the client is in
+  // @todo get this value from the app somewhere
+  USState: `MA`,              // Two-letter code denoting the state the client is in
```

Comment:
Might want to look into ESLint config option [spacing?](https://eslint.org/docs/rules/key-spacing#align) to fix this. I'm reluctant to disable ESlint for the whole file; at most, I hope we could disable just a specific rule to fix this, preferably in a block (`/* eslint-disable */ ... /* eslint-enable */`) rather than whole-file.

Reply:
The spacing rules are the thing that's causing the problem. I didn't find any spacing rules that would be more useful in there. I disabled them in this block.

Reply:
Can you disable only the rule that's causing the problem in that block, rather than a total disabling of ESLint? Also add a comment explaining why it's being disabled in that block?

Reply:
Ok, I've found a way to disable for a specific rule. I'm going to change this up, though, and disable the rule for the whole object. If/when new values are added, I don't want them to have the same unexpected result as before and have to figure it out all over again.

This also makes me question whether this column thing should be around at all. Maybe we remove the column headings and just go back to regular spacing rules. Thoughts?

Reply:
I kind of agree about the column headings--I see their value, but we could probably accomplish the same thing without them (I don't think it's too confusing what part of the code is what without the columns).

Reply:
I'll just get rid of them then.

Reply:
That's done. Is this ready for merging?